### PR TITLE
BUGS: convert document to markdown

### PR DESCRIPTION
**Pull request by bagder**

### CODE REVIEW

File: docs/BUGS.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
+                                  _   _ ____  _
+                              ___| | | |  _ \| |
+                             / __| | | | |_) | |
+                            | (__| |_| |  _ <| |___
+                             \___|\___/|_| \_\_____|
+
```
Comment (emilengler) [inline docs/BUGS.md:6]: I would suggest to either remove the ASCII art or put it into a codeblock by three back ticks.

**Review by emilengler — APPROVED** [review]

File: docs/BUGS.md (hunk 1 above)
Comment (bagder) [reply docs/BUGS.md:6]: But why? lines prefixed with 4 spaces get shown as "code" already. [see here](https://github.com/curl/curl/blob/08046135919b4334a06b16fc03649e3e50944bcf/docs/BUGS.md)

Comment (bagder) [reply docs/BUGS.md:6]: I think I should just get rid of it anyway....

Comment (emilengler) [reply docs/BUGS.md]: > But why? lines prefixed with 4 spaces get shown as "code" already. [see here](https://github.com/curl/curl/blob/08046135919b4334a06b16fc03649e3e50944bcf/docs/BUGS.md)

That's why I also approved. It was a non blocking stylish thing :)

**Pull request closed without merging**